source/frontend/src/components/maps/leaflet/Layers/Spiderfier.ts

```diff
@@ -78,7 +94,9 @@ export class Spiderfier {
     }
     this.unspiderfy();
     this.cluster = cluster;
-    const centerLatlng = GeoJSON.coordsToLatLng(cluster?.geometry?.coordinates as [number, number]);
+    const clusterX = cluster?.geometry?.coordinates[0];
```

Comment:
not sure if this is equivalent. The old method would assume an array of size 2 and operate on it, but this specifically tries to dereference index 0 and 1. The old way might not crash if the array doesn't have at least two values but this will.